homeassistant/components/evohome/manifest.json

```diff
@@ -3,7 +3,8 @@
   "name": "Evohome",
   "documentation": "https://www.home-assistant.io/components/evohome",
   "requirements": [
-    "evohomeclient==0.3.3"
+    "evohomeclient==0.3.3",
+    "python-dateutil==2.8.0"
```

Comment:
remove this and use our timezone function from `utils`

Reply:
As I indicated [above](https://github.com/home-assistant/home-assistant/pull/25010#issue-295108679), (and [here](https://github.com/home-assistant/home-assistant/pull/24491#discussion_r299011440)) I could not work out how to get the functionality I required from `utils.dt` (nor from `pytz`).

What I need to do is determine the offset used by the client library (i.e. used by the host system), which uses local/naive `datetime` objects.

In  `utils.dt`, I _can_ `get_time_zone()` for a given TZ string (e.g. `"Europe\London"`): 
```python
def get_time_zone(time_zone_str: str) -> Optional[dt.tzinfo]:
```
But how would I know right zone name, and what about daylight saving times?

I can also get the TZ for an aware `datetime` using`dt.tzinfo`. But my `datetime` objects are naive.

So I use `dateutil.tzlocal()`:
```python
def _local_dt_to_utc(dt_naive: datetime) -> datetime:
    dt_aware = as_utc(dt_naive.replace(microsecond=0, tzinfo=tzlocal()))
    return dt_aware.replace(tzinfo=None)
```